### PR TITLE
OCPCRT-272: fix OO_INDEX for sriov-network-operator

### PR DESCRIPTION
**Pull request by EmilienM**
Fix ci-operator configuration to create the bundle build.

### CODE REVIEW

**Comment by EmilienM** [comment]
/pj-rehearse pull-ci-openshift-sriov-network-operator-master-ci-index-sriov-network-operator-bundle pull-ci-openshift-sriov-network-operator-release-4.13-e2e-openstack-nfv-hwoffload

File: ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-master.yaml [hunk 1]
```diff
@@ -53,7 +57,11 @@ images:
   to: sriov-network-webhook
 operator:
   bundles:
-  - dockerfile_path: bundleci.Dockerfile
+  - as: sriov-network-operator-bundle
+    base_index: operator-index
+    context_dir: config
```
Comment (zeeke) [inline ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-master.yaml:62]: `bundleci.Dockerfile` is on the repository [root](https://github.com/openshift/sriov-network-operator/blob/master/bundleci.Dockerfile)

```suggestion
```

**Comment by EmilienM** [comment]
/pj-rehearse pull-ci-openshift-sriov-network-operator-master-ci-index-sriov-network-operator-bundle pull-ci-openshift-sriov-network-operator-release-4.13-e2e-openstack-nfv-hwoffload

**Comment by EmilienM** [comment]
2 CI jobs passed fine with this change, I believe this is ready!

File: ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-master.yaml [hunk 2]
```diff
@@ -11,6 +11,10 @@ base_images:
     name: "4.15"
     namespace: ocp
     tag: openstack-installer
+  operator-index:
+    name: redhat-operator-index
+    namespace: ci
+    tag: v4.10
```
Comment (SchSeba) [inline ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-master.yaml:17]: should you change the tag? this points to master should it be 4.15

File: ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-release-4.11.yaml [hunk 3]
```diff
@@ -11,6 +11,10 @@ base_images:
     name: "4.11"
     namespace: ocp
     tag: openstack-installer
+  operator-index:
+    name: redhat-operator-index
+    namespace: ci
+    tag: v4.10
```
Comment (SchSeba) [inline ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-release-4.11.yaml:17]: same as above

File: ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-release-4.12.yaml [hunk 4]
```diff
@@ -11,6 +11,10 @@ base_images:
     name: "4.12"
     namespace: ocp
     tag: openstack-installer
+  operator-index:
+    name: redhat-operator-index
+    namespace: ci
+    tag: v4.10
```
Comment (SchSeba) [inline ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-release-4.12.yaml:17]: same as above

File: ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-release-4.13.yaml [hunk 5]
```diff
@@ -11,6 +11,10 @@ base_images:
     name: "4.13"
     namespace: ocp
     tag: openstack-installer
+  operator-index:
+    name: redhat-operator-index
+    namespace: ci
+    tag: v4.10
```
Comment (SchSeba) [inline ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-release-4.13.yaml:17]: same as above

File: ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-release-4.14.yaml [hunk 6]
```diff
@@ -11,6 +11,10 @@ base_images:
     name: "4.14"
     namespace: ocp
     tag: openstack-installer
+  operator-index:
+    name: redhat-operator-index
+    namespace: ci
+    tag: v4.10
```
Comment (SchSeba) [inline ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-release-4.14.yaml:17]: same as above

File: ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-release-4.15.yaml [hunk 7]
```diff
@@ -11,6 +11,10 @@ base_images:
     name: "4.15"
     namespace: ocp
     tag: openstack-installer
+  operator-index:
+    name: redhat-operator-index
+    namespace: ci
+    tag: v4.10
```
Comment (SchSeba) [inline ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-release-4.15.yaml:17]: same as above

File: ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-release-4.16.yaml [hunk 8]
```diff
@@ -11,6 +11,10 @@ base_images:
     name: "4.16"
     namespace: ocp
     tag: openstack-installer
+  operator-index:
+    name: redhat-operator-index
+    namespace: ci
+    tag: v4.10
```
Comment (SchSeba) [inline ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-release-4.16.yaml:17]: same as above

File: ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-master.yaml (hunk 2 above)
Comment (EmilienM) [reply ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-master.yaml:17]: no, I checked other operators and they all use 4.10 for some reason. I left it like this for consistency.

Comment (SchSeba) [reply ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-master.yaml:17]: sounds good thanks! +1

**Comment by SchSeba** [comment]
spoke internally we can leave the version 4.10

**Pull request merged**